### PR TITLE
Ncgames 4

### PR DESCRIPTION
**Pull request by bakrikhalifa**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,16 +25,18 @@
     "express": "^4.18.2",
     "pg": "^8.7.3",
     "pg-format": "^1.0.4",
+    "sorter": "^2.0.0",
```
Comment (k-huggs) [inline package.json]: what's this dependency?

File: app.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 const express = require("express");
 const app = express();
-const { getCategories } = require("./controllers/controller");
+const { getCategories, getReviews } = require("./controllers/controller");
 
 app.get("/api/categories", getCategories);
 
+app.get("/api/reviews", getReviews);
+
 app.all("*", (req, res, next) => {
```
Comment (k-huggs) [inline app.js]: you don't need next with this piece of middleware

File: app.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
...
   res.status(404).send({ msg: "Path not found" });
 });
```
Comment (k-huggs) [inline app.js:15]: add a 500 server error handler

**Review by k-huggs — COMMENTED** [review]
See comments

File: package.json (hunk 1 above)
Comment (bakrikhalifa) [reply package.json]: I installed the wrong dependancy, was meant to install jest-sorted

File: app.js (hunk 2 above)
Comment (bakrikhalifa) [reply app.js]: Thank you will ammend

File: app.js (hunk 3 above)
Comment (bakrikhalifa) [reply app.js:15]: Will do 👍

**Pull request merged**